test/csuite/wt6616_checkpoint_oldest_ts/main.c

```diff
@@ -355,7 +356,8 @@ main(int argc, char *argv[])
         testutil_die(errno, "parent chdir: %s", home);
 
     /* Copy the data to a separate folder for debugging purpose. */
-    testutil_copy_data(home);
+    testutil_assert_errno(getcwd(buf, sizeof(buf)) != NULL);
```

Comment:
WDYT about using more descriptive name for this variable? E.g. `current_dir`?

Reply:
This functionality has been moved into the `testutil` functions so it is no longer needed here. We now save the starting cwd in the `cwd_start`. This is clearer than using `buf` and for consistency with other csuite tests